taskcluster/config.yml

```diff
@@ -1,8 +1,8 @@
 ---
 trust-domain: "translations"
 task-priority: low
-task-deadline-after: 60 days
-task-expires-after: 90 days
+task-deadline-after: 20 days
+task-expires-after: 360 days
```

Comment:
This is fine. However, this doesn't actually seem to be used at the moment though? As far as I can tell, tasks are expiring after 1y by default, so I'm not sure it will do anything...

Reply:
Oh, this is a bug in taskgraph, I think. In any case, I can see why everything is set to 1y already.